### ISSUE TITLE
No d3 format available for 'bubble size' option in bubble chart. 

### ISSUE THREAD

**Issue by cass-simplelegal** [apache/superset#23052]
## Screenshot

![ss-screenshot](https://user-images.githubusercontent.com/116036593/217840150-6dea1c04-3616-4976-9dd8-c672df3ca500.png)

<img width="1492" alt="Screen Shot 2023-02-09 at 9 27 25 AM" src="https://user-images.githubusercontent.com/116036593/217840505-e55b3126-dc39-45df-8ef0-158be9b7bd42.png">

## Description

No d3 format available for 'bubble size' option in bubble chart. When I hober over the bubble chart, I want to see MAtter days in the ",d" format. However, there is no option for this in the explore chart page, just options to format the x-axis and y-axis. 

I have also tried creating a metric and setting the fomat as ",d" but it still doesn't change the format that shows on the hover.

## Design input
Does anyone know how to get around this? Or is it a bug? 

Thanks in advance!

**Comment by rusackas** [comment]
I don't think we should consider this a bug, but more of a (perfectly reasonable) feature request. I'll convert this to an "Ideas" discussion, and I hope we can get anyone from the community to take a crack at opening a PR for it. If anyone can do so, please link it here, and I/we will be happy to review it!
